### PR TITLE
Add riot video

### LINKED ISSUE #395: Add RIOT Summit video
I have been made aware of this video from the RIOT summit 2022. We should add it:

https://www.youtube.com/watch?v=PuhNX0QdvYI&list=PLDXXQJiSjPKHyo7D9xDLJELFQw5GHKQYJ&index=2

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #395

### CODE REVIEW

File: docs/videos/index.html [hunk 1]
```diff
@@ -96,26 +96,33 @@ <h4>Short Introduction to Web of Things
     <h3>Interviews, Meetings, Conferences</h3>
     <br />
     <div class="row">
+        <div class="col-md-6">
+            <a target="_blank" href="https://www.youtube.com/watch?v=PuhNX0QdvYI&list=PLDXXQJiSjPKHyo7D9xDLJELFQw5GHKQYJ&index=3">
+                <img class="img-responsive" style="min-width: 180px; min-height: 245px; width: 100%; border-width: thin; border-style: solid;"
+                    src="{{'/images/video-thumbnail-riot-2022.png' | absolute_url }}" alt="WoT Video" />
+                <h4>RIOT Summit 2022, Keynote: "Web of Things - What is it all about"</h4>
+                <!-- 2021-11-05 -->
```
Comment (danielpeintner) [inline docs/videos/index.html:104]: I guess this comment about the date is not correct

Comment (egekorkan) [reply docs/videos/index.html:104]:
```suggestion
                <!-- 2022-09-05 -->
```

Comment (egekorkan) [reply docs/videos/index.html]: fixed :)

**Review by danielpeintner — APPROVED** [review]

**Comment by egekorkan** [comment]
Call of 16.05: We will merge but then remove the min-height etc. attributes as a follow-up at https://github.com/w3c/wot-marketing/issues/413

**Pull request merged**